src/screens/MainApp/DetailedDiagnose/components/Comment/index.js

```diff
@@ -1,16 +1,37 @@
-import React from 'react'
-import { View } from 'react-native'
+import React, { useCallback } from 'react'
+import { View, FlatList } from 'react-native'
+import NavigationService from '~/navigationService'
 import { Subheader, Body, Date } from '~/components/texts'
+import ImageButton from '~/components/buttons/ImageButton'
 import styles from './styles'
 
-const Comment = ({ text, by, date }) => {
+const Comment = ({ text, by, date, images }) => {
+  const renderImage = useCallback((image, index) => {
+    return (
+      <ImageButton
+        onPress={() => NavigationService.navigate('FullScreenImagesView', { images, startImage: index, title: by })}
+        source={{ uri: image }}
+        style={styles.image}
+      />
+    )
+  }, [])
+
   return (
     <View style={styles.commentContainer}>
       <View style={styles.metadataContainer}>
         <Subheader style={styles.by}>{by}</Subheader>
         <Date gray2>{date}</Date>
       </View>
       <Body gray>{text}</Body>
+      {images && (
+        <FlatList
+          data={images}
+          horizontal
+          showsHorizontalScrollIndicator={false}
+          renderItem={({ item, index }) => renderImage(item, index)}
```

Comment:
By using a lambda function here, you're throwing away all the benefits of using `useCallback` to define `renderImage` :smile: Just destructure the parameters in the callback definition above and pass `renderImage` directly here.